practitioner-service/src/main/java/uk/ac/ox/ndph/arts/practitioner_service/model/Person.java

```diff
@@ -0,0 +1,25 @@
+package uk.ac.ox.ndph.arts.practitioner_service.model;
+
+import java.io.Serializable;
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.RequiredArgsConstructor;
+import lombok.NonNull;
+import lombok.NoArgsConstructor;
+
+@Getter
+@Setter
+@NoArgsConstructor
+public class Person implements Serializable {
+
+    private String prefix;
+    private String givenName;
+    private String familyName;
+
+    public Person(String prefix, String givenName, String familyName) {
```

Comment:
This is actually an @AllArgsConstructor, so can be dropped as well. Is it required for constructing the object from the application or for Jackson serde operations?